rhel-8.5.yaml

```diff
@@ -0,0 +1,177 @@
+# Layers specific to RHCOS
+ostree-layers:
+  # Temporary logrotate service and timer units
+  - overlay/15rhcos-logrotate
+
+# See README.md
+# and https://github.com/openshift/release/blob/master/core-services/release-controller/README.md#rpm-mirrors
+repos:
+  - rhel-8-baseos
+  - rhel-8-appstream
+  - rhel-8-fast-datapath
+  - rhel-8-server-ose
+
+# https://bugzilla.redhat.com/show_bug.cgi?id=1938928
+rpmdb: bdb
+
+# We include hours/minutes to avoid version number reuse
+automatic-version-prefix: "411.85.<date:%Y%m%d%H%M>"
+# This ensures we're semver-compatible which OpenShift wants
+automatic-version-suffix: "-"
+# Keep this is sync with the version in postprocess
+mutate-os-release: "4.11"
+
+postprocess:
+  - |
+     #!/usr/bin/env bash
+     set -xeo pipefail
+
+     # Tweak /usr/lib/os-release
```

Comment:
Can't all this stuff be in `common.yaml` too?

Reply:
It depends on how much we want to change things here for SCOS. Right now it's a copy paste for SCOS but maybe it should not.